res/skins/QMLDemo/Button.qml

```diff
@@ -0,0 +1,128 @@
+import "." as Skin
+import QtGraphicalEffects 1.12
+import QtQml 2.12
+import QtQuick 2.12
+import QtQuick.Controls 2.12
+import "Theme"
+
+AbstractButton {
+    id: root
+
+    property color normalColor: Theme.buttonNormalColor
+    required property color activeColor
```

Comment:
Because buttons may have different active colors depending on context. E. g. PFL (Mixer) is blue, but CUE (Deck) is green.

Reply:
Right, but if they just want to use a default, the have to specify that explicitly instead of the button falling back on the Theme.

Reply:
Correct. I there is no sensible default, so making it required to specify ensures I don't forget it and the skin looking bad.

Reply:
Is there really no sensible default for the active color of a button? Couldn't the active color just default to `Qt.lighter(normalColor, 1.5)`?

Reply:
Or just have a active default color in the Theme?

Reply:
I mean this is just a non-reusable skin component.

Alao, In c++ code we usually avoid default arguments when not needed, so I don't really understand why we should add it here.

Can you elaborate why this is needed?

Reply:
I agree, I'm sorry I'm nagging so much about the polish of a prototype.

I'd say its needed because it removed redundancy in the skin (technically, not sure if this is actually an annoyance, I didn't write the code ;) )

What does @ronso0 think about this?

Reply:
No problem, I'm not annoyed :D I just wondered why you think it's necessary/helpful.